src/BottomNavigation/BottomNavigation.react.js

```diff
@@ -44,20 +42,67 @@ function getStyles(props, context) {
 */
 
 class BottomNavigation extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            moveAnimated: new Animated.Value(0 - this.context.uiTheme.bottomNavigation.container.height),
```

Comment:
Yeah, I have it configured to animate in at mount, but I think it's more appropriate to init it in view. That will mean there is no need for the context in the constructor.